ksql-streams/src/main/java/io/confluent/ksql/execution/streams/materialization/ks/KsMaterializedSessionTable.java

```diff
@@ -79,7 +79,14 @@
               Optional.of(next.key.window().endTime())
           );
 
-          builder.add(WindowedRow.of(stateStore.schema(), key, window, next.value));
+          final WindowedRow row = WindowedRow.of(
+              stateStore.schema(),
+              key, window,
+              next.value,
+              next.key.window().end()
```

Comment:
I'm guessing we're using the key window end as the value for rowtime? can we add a comment explaining that semantic? is that what we want?